docs/source/quickstart/index.rst

```diff
@@ -0,0 +1,216 @@
+==================================
+Welcome to Avocado's Documentation
+==================================
+
+Avocado is a set of tools and libraries to help with automated testing.
+
+One can call it a test framework with benefits.  Native tests are written in
+Python and they follow the :mod:`unittest` pattern, but any executable can
+serve as a test.
+
+How it works?
+=============
+
+You should first experience Avocado by using the test runner, that is, the
+command line tool that will conveniently run your tests and collect their
+results.
+
+To do so, please run ``avocado`` with the ``run`` sub-command followed by a
+test reference, which could be either a path to the file, or a recognizable
+name::
+
+    $ avocado run /bin/true
+    JOB ID     : 3a5c4c51ceb5369f23702efb10b4209b111141b2
+    JOB LOG    : $HOME/avocado/job-results/job-2019-10-31T10.34-3a5c4c5/job.log
+     (1/1) /bin/true: PASS (0.04 s)
+    RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
+    JOB TIME   : 0.15 s
+
+You probably noticed that we used ``/bin/true`` as a test, and in accordance
+with our expectations, it passed! These are known as `simple tests`, but there
+is also another type of test, which we call `instrumented tests`. See more at
+`test-types` or just keep reading.
+
+Why should I use?
+=================
+
+Multiple result formats
+-----------------------
+
+A regular run of Avocado will present the test results on standard output, a
+nice and colored report useful for human beings. But results for machines can
+also be generated.
+
+Check the job-results folder (``$HOME/avocado/job-results/latest/``) to see the
+outputs.
+
+Currently we support, out of box, the following output formats:
+
+  * **xUnit**: an XML format that contains test results in a structured form,
+    and are used by other test automation projects, such as jenkins.
+
+  * **JSON**: a widely used data exchange format. The JSON Avocado plugin
+    outputs job information, similarly to the xunit output plugin.
+
+  * **TAP**: Provides the basic TAP (Test Anything Protocol) results,
+    currently in v12. Unlike most existing avocado machine readable outputs
+    this one is streamlined (per test results).
+
+.. note:: You can see the results of the lastest job inside the folder
+  ``$HOME/avocado/job-results/latest/``. You can also specify at the command line
+  the options ``--xunit``, ``--json`` or ``--tap`` followed by a filename.
+  Avocado will write the output on the specified filename.
+
+When it comes to outputs, avocado is very flexible. You can check the various
+**output plugins**. If you wanted something more sophisticated, visit our
+plugins section.
+
+Sysinfo data collector
+----------------------
+
+Avocado comes with a sysinfo plugin, which automatically gathers some system
+information per each job or even between tests. This is very useful when later
```

Comment:
Suggestion: "This is very helpful when trying to identify the cause of a test failure".